scipy/optimize/_trustregion_constr/tests/test_qp_subproblem.py

```diff
@@ -558,7 +558,7 @@ def test_inactive_box_constraints(self):
         assert_array_almost_equal(x, x_kkt)
 
     # The box constraints active and the termination is
-    # by maximum iterations (infeasible iteraction).
+    # by maximum iterations (infeasible iteration).
```

Comment:
This and below should almost certainly be "interaction"

Reply:
My wrong. I had to choose between these two words, and I have to admit I don't understand the comment – I probably lack context.